### PR TITLE
refactor(ui): main flow ui refine part 2

### PR DESCRIPTION
**Pull request by simeng-li**
### main flow ui refine 2nd round

1. Update the divider Or content to lowercase
image

2. Set the input field container height fixed with overflow hidden. As in safari input field,[ line-height is differ](https://stackoverflow.com/questions/42529257/input-text-height-is-broken-in-safari) 

3. Update the font-family settings that fix the passcode secret mast dot size and styling
image

5. Disable all elements on-touch event shadow effect
6. Update the modal padding space
7. Adjust the text link underline space
image

9. Update the NavBar icon fill color use the base text color
image

10. Add border-radius to the drawer element
image

11. Prevent body scall when modal occurred
12. Adjust the link size within hint text content.
13. Callback Page layout redesign
image

## Linear Issue Reference

LOG-2367

## Testing

test locally
@logto-io/eng

### CODE REVIEW

File: packages/ui/src/components/ConfirmModal/index.tsx [hunk 1]
```diff
@@ -37,6 +37,14 @@ const ConfirmModal = ({
...
       parentSelector={() => document.querySelector('main') ?? document.body}
       ariaHideApp={false}
+      onAfterOpen={() => {
+        /* eslint-disable @silverhand/fp/no-mutation */
+        document.body.style.overflow = 'hidden';
+      }}
+      onAfterClose={() => {
+        document.body.style.overflow = 'unset';
+        /* eslint-enable @silverhand/fp/no-mutation */
+      }}
```
Comment (demonzoo) [inline packages/ui/src/components/ConfirmModal/index.tsx:47]: Is it possible to use CSS? For example:
```typescript
document.body.classList.add(className);
```

File: packages/ui/src/components/Drawer/index.tsx [hunk 2]
```diff
@@ -26,6 +26,14 @@ const Drawer = ({ className, isOpen = false, children, onClose }: Props) => {
...
       closeTimeoutMS={300}
       onRequestClose={onClose}
+      onAfterOpen={() => {
+        /* eslint-disable @silverhand/fp/no-mutation */
+        document.body.style.overflow = 'hidden';
+      }}
+      onAfterClose={() => {
+        document.body.style.overflow = 'unset';
+        /* eslint-enable @silverhand/fp/no-mutation */
+      }}
```
Comment (demonzoo) [inline packages/ui/src/components/Drawer/index.tsx:36]: Same here. Would be nice if we can use CSS

File: packages/ui/src/components/ConfirmModal/index.tsx (hunk 1 above)
Comment (IceHe) [inline packages/ui/src/components/ConfirmModal/index.tsx:47]: nit: if not use css, maybe better
```suggestion
      /* eslint-disable @silverhand/fp/no-mutation */
      onAfterOpen={() => {
        document.body.style.overflow = 'hidden';
      }}
      onAfterClose={() => {
        document.body.style.overflow = 'unset';
      }}
      /* eslint-enable @silverhand/fp/no-mutation */
```

File: packages/ui/src/components/Drawer/index.tsx (hunk 2 above)
Comment (IceHe) [inline packages/ui/src/components/Drawer/index.tsx:36]: nit: if not use css, ditto

File: packages/ui/src/components/ConfirmModal/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/ui/src/components/ConfirmModal/index.tsx:47]: Or add some comments?

Comment (simeng-li) [reply packages/ui/src/components/ConfirmModal/index.tsx:47]: Adding classname would require a global classname defined, not sure which one is more straightforward?

Comment (simeng-li) [reply packages/ui/src/components/ConfirmModal/index.tsx:47]: I'll go with the classname then

Comment (simeng-li) [reply packages/ui/src/components/ConfirmModal/index.tsx]: @wangsijie @demonzoo updated PTAL

**Review by wangsijie — APPROVED** [review]

**Pull request merged**